Library/Homebrew/utils/curl.rb

```diff
@@ -44,12 +44,12 @@ def curl_args(*extra_args, show_output: false, user_agent: :default)
   args + extra_args
 end
 
-def curl(*args, secrets: [], **options)
+def curl(*args, print_stdout: true, secrets: [], **options)
```

Comment:
Is this still needed or can it be reverted?

Reply:
Can be reverted!